active/0000-blueprint-remove-old-files.md

```diff
@@ -0,0 +1,50 @@
+- Start Date: 2016-12-17
+- RFC PR: (leave this empty)
+- Ember CLI Issue: (leave this empty)
+
+# Summary
+
+Give blueprint generators the ability to clean up old files.
+
+# Motivation
+
+We want to eliminate the noise of having old files laying after updating
+ember-cli using `ember init`.
+
+# Detailed design
+
+We'd like an API for blueprints to delete files instead of only
+create. It would be essentially syntactic sugar for removing the file yourself
+in an `afterInstall` hook. It would be a returned array on the blueprint's `index.js`.
```

Comment:
So today, file mutations are intended to be transactional. Specifically, it is possible cancel / say no abort a blueprint and abort. As described, this appears like it may not participate in that, is that the case?